packages/uniforms/__suites__/ListDelField.tsx

```diff
@@ -0,0 +1,76 @@
+import { screen } from '@testing-library/react';
+import userEvent, {
+  PointerEventsCheckLevel,
+} from '@testing-library/user-event';
+import React, { ComponentType } from 'react';
+import z from 'zod';
+
+import { renderWithZod } from './render-zod';
+
+export function testListDelField(ListDelField: ComponentType<any>) {
+  test('<ListDelField> - correctly reacts on click', async () => {
+    const onChange = jest.fn();
+    renderWithZod({
+      element: <ListDelField name="x.1" data-testid="x" />,
+      model: { x: ['a', 'b', 'c'] },
+      onChange,
+      schema: z.object({ x: z.array(z.string()) }),
+    });
+    await userEvent.click(screen.getByTestId('x'));
+    expect(onChange).toHaveBeenLastCalledWith('x', ['a', 'c']);
+  });
+
+  test('<ListDelField> - correctly reacts on enter', async () => {
+    const onChange = jest.fn();
+    renderWithZod({
+      element: <ListDelField name="x.1" data-testid="x" />,
+      model: { x: ['a', 'b', 'c'] },
+      onChange,
+      schema: z.object({ x: z.array(z.string()) }),
+    });
+    await userEvent.type(screen.getByTestId('x'), '{Enter}');
```

Comment:
Not sure if this is a typo or not, [but in the documentation](https://testing-library.com/docs/ecosystem-user-event/#special-characters), it's lowercase `{enter}`. No idea if this has any impact.